src/deploy/functions/params.ts

```diff
@@ -30,7 +30,7 @@ export function resolveInt(
   from: number | build.Expression<number>,
   paramValues: Record<string, build.Field<string | number | boolean>>
 ): number {
-  if (typeof from === "number") {
+  if (typeof from === "number" || typeof from === "undefined") {
```

Comment:
Consider just doing `from === undefined`, which _should_ give you the same behavior (but seem a little less clunky/extra-stepy)

Reply:
`undefined` is a variable, not a constant. JavaScript best practices is always to compare `typeof`.

Reply:
Whelp, my overly pedantic coding style is based on an outdated concern (and one which would almost certainly not happen w/o a malicious NPM module). ICIMIY: `undefined` is technically a variable, not a constant. So in the past you could assign `undefined = 5` and things that had a `typeof (x) === "undefined"` would not `== undefined`.

When I tried to reproduce this insanity though it seems like Node fixed this by _still allowing_ assignment to undefined, but making it not do anything.